store/postgres/src/sql_value.rs

```diff
@@ -53,3 +53,12 @@ impl ToSql<Text, Pg> for SqlValue {
         }
     }
 }
+
+impl ToSql<Binary, Pg> for SqlValue {
+    fn to_sql<W: Write>(&self, out: &mut Output<W, Pg>) -> serialize::Result {
+        match self.0 {
+            Value::Bytes(ref h) => <_ as ToSql<Binary, Pg>>::to_sql(&h.as_slice(), out),
+            _ => panic!("Failed to convert attribute value to String or Bytes in SQL"),
```

Comment:
Yeah, I wondered that, too, but this is just copied from what we've always done for all of the other `ToSql<_, _>` implementations. My rationale for a `panic!` would be that hitting it constitutes a programming error, and it's more likely this gets caught in testing if we panic rather than reporting an error. Either way, if you want it changed, can you open an issue for it? I'd rather address that separately.